apps/aecore/oracle.py

```diff
@@ -0,0 +1,39 @@
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+import SocketServer
+import requests
+import json
+import urllib2
+import thread
+
+class S(BaseHTTPRequestHandler):
+    def _set_headers(self):
+        self.send_response(200)
+        self.send_header('Content-type', 'application/json')
+        self.end_headers()
+
+    def do_HEAD(self):
+        self._set_headers()
+
+    def do_POST(self):
+        content_length = int(self.headers['Content-Length'])
+        post_data = json.loads(self.rfile.read(content_length))['data']
+        get_data = json.loads(urllib2.urlopen("https://api.fixer.io/latest?symbols=" + post_data['query_data']['currency']).read())
+        payload = {"oracle_hash":post_data['oracle_hash'], "response":get_data, "fee":10}
+        req = urllib2.Request('http://localhost:4000/oracle_response')
```

Comment:
I have trouble understanding the purpose of this python http server. Also:
- it has hardcoded node port
- it is open to the word (?)
- it has some wird hardcoded call to "api.fixer.io"

Reply:
it is a sample oracle data providing service, for merging, we should remove this @d-velev

Reply:
The oracle test uses it